main.py

```diff
@@ -45,38 +45,62 @@ def distance(origin, destination):
 asdex_px_per_nm = screen_height / distance((airport_data["asdex_top"], 0), (airport_data["asdex_bottom"], 0))
 radar_px_per_nm = screen_height / distance((airport_data["top"], 0), (airport_data["bottom"], 0))
 scene = True
+debug = False
 
 pygame.init()
 clock = pygame.time.Clock()
 screen = pygame.display.set_mode((screen_width, screen_height))
 
 class Aircraft(pygame.sprite.Sprite):
-    def __init__(self, loc, speed, heading):
+    def __init__(self, loc, speed, heading, name, altitude):
         pygame.sprite.Sprite.__init__(self)
-        self.surf = pygame.Surface((75, 15), pygame.SRCALPHA)
-        pygame.draw.rect(self.surf, (86, 176, 91), (0, 7.5, 5, 5))
+        self.name = name
+        self.surf = pygame.Surface((750, 15), pygame.SRCALPHA)
+        # pygame.draw.rect(self.surf, (86, 176, 91), (0, 7.5, 5, 5))
```

Comment:
If lines 59 and 62 are needed on init, we can add them back, but it didn't seem to hurt anything to remove it